service/round.go

```diff
@@ -332,12 +337,12 @@ func (r *round) calcMembersAndStatement() ([][]byte, []byte, error) {
 	iter := r.challengesDb.Iterator()
 	defer iter.Release()
 	for iter.Next() {
-		key := iter.Key()
-		keyCopy := make([]byte, len(key))
-		copy(keyCopy, key)
+		challenge := iter.Value()
+		challengeCopy := make([]byte, len(challenge))
+		copy(challengeCopy, challenge)
```

Comment:
I don't know _for sure_, but I think it's because the slice points to some _snapshot_ of DB (accessed by the Iterator) and it might be reused/changed after the Iterator is released.